pykokkos/lib/ufuncs.py

```diff
@@ -260,13 +268,18 @@ def sign(view):
 
 @pk.workunit
 def add_impl_1d_double(tid: int, viewA: pk.View1D[pk.double], viewB: pk.View1D[pk.double], out: pk.View1D[pk.double], ):
-    out[tid] = viewA[tid] + viewB[tid]
+    out[tid] = viewA[tid] + viewB[tid % viewB.extent(0)]
```

Comment:
Is this needed if viewA and viewB have different sizes? Is this what numpy does?

Reply:
I think numpy does not do this. Since we do not have a mechanism in place to verify broadcasting, this will work for now assuming the inputs can be broadcasted.
Another option could be to make separate workunits for all the different combinations of inputs. Please let me know if you think we should go down that route instead.